setup.cfg

```diff
@@ -1,4 +1,4 @@
 [develop]
-script-dir=$base/share/urdfdom_py/scripts
+script_dir=$base/share/urdfdom_py/scripts
 [install]
-install-scripts=$base/share/urdfdom_py/scripts
+install_scripts=$base/share/urdfdom_py/scripts
```

Comment:
This is different from https://github.com/ros2/examples/blob/master/rclpy/executors/setup.cfg where scripts are installed into `$base/lib/package name`. I think that's the only place ros2 looks for scripts to run.

Reply:
That is an excellent point, and indeed no binaries are found with `ros2 run urdfdom_py display_urdf`.  I'll merge this one in and also open another PR to fix that.

Reply:
where can I find this `setup.cfg` file?